packages/react-components/priority-overflow/src/overflowManager.ts

```diff
@@ -148,6 +136,16 @@ export function createOverflowManager(): OverflowManager {
     return offsetSize;
   };
 
+  const getClientSize = (el: HTMLElement) => {
+    if (sizeCache.has(el)) {
```

Comment:
is it necessary to cache this? I only see the `clientSize` used once per update to get the available size of the container

Reply:
oh, I need to refactor this function. I forgot about this

Reply:
I refactored it. But caching is there for consistency